device-types/Fortinet/FG-2200E.yaml

```diff
@@ -0,0 +1,98 @@
+---
+manufacturer: Fortinet
+model: FortiGate 2200E
+slug: fg-2200e
+part_number: FG-2200E
+u_height: 2
+is_full_depth: false
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
```

Comment:
Are these modular/hot swappable? If so, you should remove the `power-ports` definitions, add `module-bays` definition and create the applicable PSUs in the module-types folder if they do not already exist

Reply:
Yes, there are hot swappable, but after I can't find the information on how to model them correctly: I'm missing the model name and the part number (at least one of them). I'll remove the power ports in the device types.

Reply:
A good example of how to model the PSU module type is: (you need the model/part number though_

```
---
manufacturer: Cisco
model: UCS-PSU-6332-AC
part_number: UCS-PSU-6332-AC
power-ports:
  - name: PSU{module}
    type: iec-60320-c14
    maximum_draw: 650
```

Reply:
I know how to do it but I'm missing the model and part number.